tests/core/gui/test_create_organization.py

```diff
@@ -278,3 +284,88 @@ def _modal_shown():
         assert autoclose_dialog.dialogs == [("Error", "This bootstrap link was already used.")]
 
     await aqtbot.wait_until(_modal_shown)
+
+
+@pytest.mark.gui
+@pytest.mark.trio
+@customize_fixtures(backend_spontaneous_organization_boostrap=True)
+async def test_create_organization_custom_backend(
+    gui, aqtbot, running_backend, catch_create_org_widget, autoclose_dialog
+):
+    # The org creation window is usually opened using a sub-menu.
+    # Sub-menus can be a bit challenging to open in tests so we cheat
+    # using the keyboard shortcut Ctrl+N that has the same effect.
+    await aqtbot.key_clicks(gui, "n", QtCore.Qt.ControlModifier)
+
+    co_w = await catch_create_org_widget()
+    co_w.config = co_w.config.evolve(
+        preferred_org_creation_backend_addr=BackendAddr.from_url("parsec://localhost:1337")
```

Comment:
Providing a dummy port on localhost should be done with the `unused_tcp_port` to ensure the port is really not in use.

Beside I think we should think of a better way to patch the config here: the config is supposed to be immutable so it would be better to first craft a config, then use to create the gui (this is usually done with the `customize_fixtures` decorator).

And last but not least, `preferred_org_creation_backend_addr` field from the `core_config` is already configured to be the `backend_addr` fixture. This is good because we don't risk any request to external addresses and we can use the offline system to achieve our goal "with rare elegance"®:
```python
@pytest.fixture
def preferred_org_creation_backend_addr(backend_addr):
    return addr_with_device_subdomain(backend_addr, 'preferred_org_creation')  # parsec://preferred_org_creation.example.com:9999?use_ssl=false

@pytest.fixture
def core_config(tmpdir, preferred_org_creation_backend_addr):
    # We must provide a custom backend address with a subname here given offline(backend_addr) would put offline all subdomain
    tmpdir = Path(tmpdir)
    return CoreConfig(
        [...]
        preferred_org_creation_backend_addr=preferred_org_creation_backend_addr,
    )

def tost_turbo_stuffing_the_stuff(preferred_org_creation_backend_addr, backend_addr):
    custom_addr = addr_with_device_subdomain(backend_addr, 'custom')  # parsec://custom.example.com:9999?use_ssl=false
    with offline(preferred_org_creation_backend_addr):
        # From now on, only preferred_org_creation.example.com:9999 is offline, so we can use custom.example.com:9999
        assert_org_creation_with_preferred_addr_doesnt_work()
        assert_org_creation_with_custom_addr_work(custom_addr)
```